dvc/command/remote.py

```diff
@@ -106,22 +106,24 @@ def run(self):
 
 class CmdRemoteRename(CmdRemote):
     def _rename_default(self, conf):
-        if conf["core"].get("remote") == self.args.name:
-            conf["core"]["remote"] = self.args.new
+        if conf["core"].get("remote") == self.args.old_name:
```

Comment:
Let's not touch unrelated code unless we really need too.

Reply:
reverted